docs/upgrade/UPGRADE-v7.3.2-dev.md

```diff
@@ -32,6 +32,10 @@ There you can find links to upgrade notes for other versions too.
         - even though both spellings are valid, "email" is much more common
     - make these replacements in your application code, test code, data fixtures, and translation messages
 
+- fix brand label when the brand name is long or have two lines ([#1371](https://github.com/shopsys/shopsys/pull/1371))
+    - update your [`src/Shopsys/ShopBundle/Resources/views/Front/Content/Brand/list.html.twig`] (https://github.com/shopsys/project-base/blob/v7.3.2/src/Shopsys/ShopBundle/Resources/views/Front/Content/Brand/list.html.twig)
```

Comment:
```suggestion
    - update your `src/Shopsys/ShopBundle/Resources/views/Front/Content/Brand/list.html.twig` according to [the file in shopsys/project-base](https://github.com/shopsys/project-base/blob/v7.3.2/src/Shopsys/ShopBundle/Resources/views/Front/Content/Brand/list.html.twig) repository
```

Reply:
Maybe it would be better show diff of this change instead of linking to the repository because its a small change. What do you think?

something like:
```diff
    <li class="list-images__item">
-       <a href="{{ url('front_brand_detail', { id: brand.id }) }}" class="list-images__item__block list-images__item__block--with-label">
-           {{ image(brand, { alt: brand.name }) }}
-           <span>{{ brand.name }}</span>
+       <a href="{{ url('front_brand_detail', { id: brand.id }) }}">
+           <div class="list-images__item__label">{{ brand.name }}</div>
+           <div class="list-images__item__image">
+               {{ image(brand, { alt: brand.name }) }}
+           </div>
        </a>
    </li>
```

Reply:
I think it will be batter to show all diffs or none diff